appcues/src/main/java/com/appcues/debugger/ui/DebuggerFloatingActionEvents.kt

```diff
@@ -74,7 +75,9 @@ internal fun BoxScope.DebuggerFloatingActionEvents(
                 .offset { eventsProperties.positionOffset },
             contentAlignment = Alignment.TopEnd
         ) {
-            val shouldDisplayEvents = debuggerState.isDragging.targetState.not() && debuggerState.isExpanded.targetState.not()
+            val shouldDisplayEvents = debuggerState.debugMode.value is Debugger &&
+                debuggerState.isDragging.targetState.not() &&
+                debuggerState.isExpanded.targetState.not()
```

Comment:
only display floating events in `Debugger` mode

Reply:
will there be a way to exit capture mode without dismissing the FAB?

Reply:
Sam indicated that the only way to exit is dragging the FAB to the dismiss zone (or deep link back into debugger mode)